docs/common/error-utilities.ts

```diff
@@ -245,6 +246,11 @@ const RENAMED_PAGES: Record<string, string> = {
   '/build-reference/how-tos/': '/build-reference/private-npm-packages/',
   '/get-started/': '/get-started/installation/',
   '/guides/detach/': '/archive/glossary/#detach',
+  '/workflow/snack/': 'https://snack.expo.dev/',
```

Comment:
```suggestion
  '/workflow/snack/': 'https://snack.expo.dev/',
```

Would be nice to have a page descripting what is Snack, how and when you can use it. Only mention which I was able to find currently about it is a glossary entry:
* https://docs.expo.dev/more/glossary-of-terms/#snack

Reply:
We used to have one but during organization it was decided against it but I can see the point. After you pointed out, I thinkwe should point this to the Glossary description instead of the external link for now.